explain-index-merge.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 启用 `Explain` 语句的 `IndexMerge` 特性
```

Comment:
Using ` backticks in the metadata section causes parsing failure. Avoid using it here.
```suggestion
title: 启用 Explain 语句的 IndexMerge 特性
```

Reply:
Got it